vendor/blockchain_contracts/src/ethereum/rfc003/ether_htlc.rs

```diff
@@ -1,16 +1,15 @@
-use crate::{
-    offset_parameter::{apply_offsets, Error, OffsetParameter},
-    rfc003::{secret_hash::SecretHash, timestamp::Timestamp},
-};
+use crate::offset_parameter::{apply_offsets, OffsetParameter};
 use std::ops::Range;
-use web3::types::Address;
+use web3::types::{Address, U256};
 
 pub const SECRET_HASH_RANGE: Range<usize> = 51..83;
 pub const EXPIRY_RANGE: Range<usize> = 99..103;
 pub const REDEEM_IDENTITY_RANGE: Range<usize> = 153..173;
 pub const REFUND_IDENTITY_RANGE: Range<usize> = 214..234;
 
-const CONTRACT_TEMPLATE: & str      = "6100dc61000f6000396100dc6000f336156051576020361415605c57602060006000376020602160206000600060026048f17f0000000000000000000000000000000000000000000000000000000000000000602151141660625760006000f35b42630000000010609f575b60006000f35b7fb8cac300e37f03ad332e581dea21b2f0b84eaaadc184a295fef71e81f44a741360206000a1730000000000000000000000000000000000000000ff5b7f5d26862916391bf49478b2f5103b0720a842b45ef145a268f2cd1fb2aed5517860006000a1730000000000000000000000000000000000000000ff";
+use hex_literal::hex;
```

Comment:
:x: Please merge with the other `use` statements.

Reply:
❓ Do you mean I should remove the empty line?